### PR TITLE
Fix incorrect print commands in spreaper

### LINKED ISSUE #1282: Unable to create snapshot with spreaper
[Project board link](https://github.com/orgs/k8ssandra/projects/8/views/1?pane=issue&itemId=22086141)

Hi,
I'm unable to overcome this issue when I try to create snapshot with a spreaper from CLI in a reaper POD  (**image cassandra-reaper:latest sha256:09bf27ed1bcd28494389444d1c4be8123eff276fc48f846fbe32dfee90cceab8** )

Help says:

```
usage: spreaper [-h] [--reaper-host REAPER_HOST] [--reaper-port REAPER_PORT] [--reaper-use-ssl] [-v] [-vv] [-q] [--jwt JWT] [--node NODE] [--keyspace KEYSPACE] [--name NAME] [--owner OWNER] [--cause CAUSE] take-snapshot [cluster_name]

Take a snapshot. You need to register a cluster into Reaper (add-cluster) before calling this.

positional arguments:
  take-snapshot
  cluster_name          the cluster name
```
I have a cluster registered, so when I try to create snapshot I get this error:

```
bash-5.1$ spreaper take-snapshot <cluster-name>
# Taking a snapshot on cluster '{0}', 
Traceback (most recent call last):
  File "/usr/local/bin/spreaper", line 920, in <module>
    ReaperCLI()
  File "/usr/local/bin/spreaper", line 426, in __init__
    getattr(self, command)()
  File "/usr/local/bin/spreaper", line 618, in take_snapshot
    print ("# Taking a snapshot on cluster '{0}', " 
AttributeError: 'NoneType' object has no attribute 'format'
```
python version is 3.9.15 .

Do you have  idea what is wrong ? I have tried multiple variants of spreaper calls and parameters. 

Thanks!

### PR DESCRIPTION
**Pull request by adejanovski**
Fixes #1282

The print commands in spreaper's `take-snapshot` method (and a few others) was not built correctly and had misplaced closing parenthesis.

### CODE REVIEW

File: src/packaging/bin/spreaper [hunk 1]
```diff
@@ -594,29 +594,29 @@ class ReaperCLI(object):
...
             if args.node:
                 print ("# Taking a snapshot on cluster '{0}' and node '{1}', " 
-                        ).format(args.cluster_name, args.node)
+                        .format(args.cluster_name, args.node))
                 reply = reaper.post("snapshot/{0}/{1}".format(args.cluster_name, args.node), 
                                     keyspace=args.keyspace, 
                                     owner=args.owner, cause=args.cause,
                                     snapshot_name=args.name)
             else:
                 print ("# Taking a snapshot on cluster '{0}', "
```
Comment (Miles-Garnsey) [inline src/packaging/bin/spreaper:618]: Extra comma here.

Comment (Miles-Garnsey) [reply src/packaging/bin/spreaper:618]: This prints 

```
# Taking a snapshot on cluster 'cname', 
```

File: src/packaging/bin/spreaper [hunk 2]
```diff
@@ -638,11 +638,11 @@ class ReaperCLI(object):
         
         if args.node:
             print ("# Deleting snapshot '{1}' on cluster '{0}' and node '{2}'"
```
Comment (Miles-Garnsey) [inline src/packaging/bin/spreaper]: Please check the ordering of arguments for this one. I think you've reversed cluster name and snapshot name.

File: src/packaging/bin/spreaper [hunk 3]
```diff
@@ -725,8 +725,8 @@ class ReaperCLI(object):
             exit(1)
         if args.tables:
             print ("# Registering repair run for cluster '{0}', and keyspace '{1}', "
-                   "targeting tables: {2}").format(args.cluster_name, args.keyspace_name,
-                                                   ",".join(args.tables.split(',')))
+                   "targeting tables: {2}".format(args.cluster_name, args.keyspace_name,
+                                                   ",".join(args.tables.split(','))))
```
Comment (Miles-Garnsey) [inline src/packaging/bin/spreaper]: Why do you split and then rejoin? Seems redundant.

**Review by Miles-Garnsey — CHANGES_REQUESTED** [review]
Some requested changes...

File: src/packaging/bin/spreaper (hunk 3 above)
Comment (adejanovski) [reply src/packaging/bin/spreaper]: The hell if I know 😅 so weird...

File: src/packaging/bin/spreaper (hunk 2 above)
Comment (adejanovski) [reply src/packaging/bin/spreaper]: right 👍

File: src/packaging/bin/spreaper (hunk 1 above)
Comment (adejanovski) [reply src/packaging/bin/spreaper:618]: 👍

**Review by Miles-Garnsey — APPROVED** [review]
Looks better now!

**Pull request merged**